kedro_vertexai/cli.py

```diff
@@ -98,14 +97,6 @@ def list_pipelines(ctx):
     help="Parameters override in form of `key=value`",
 )
 @click.option("--wait-for-completion", type=bool, is_flag=True, default=False)
-@click.option(
```

Comment:
Why was timeout removed?

Reply:
In the old version of the plugin, the timeout meant only the timeout of the local python process (how long to wait for the pipeline completion until the execution of the local command was exited), not the timeout of the actual remote pipeline on Vertex AI. So even if the pipeline execution time exceeded the specified timeout value, the remote pipeline won't be altered in any way and will continue to run.
I've talked about it with @em-pe and the plan is to develop a proper timeout that would also alter the remote pipeline execution, possibly including the per-task timeout, not only the whole pipeline timeout.

Reply:
Ok, some of this info should be included in the changelog as it's an api change of usage of our plugin and should be documented. Probably docs also need updating.

Reply:
Ok, sure!